.github/workflows/deploy-to-devnet.yml

```diff
@@ -49,58 +55,52 @@ jobs:
-
-          docker run -i -v $(pwd):/app public.ecr.aws/p6e8q1z1/fork-off:latest --ws-rpc-endpoint=wss://ws.test.azero.dev --initial-spec-path=chainspec.skeleton.json --combined-spec-path=chainspec.json
+      - name: Generate chainspec
+        env:
+          RELEASE_TAG: ${{ steps.vars.outputs.sha_short }}
 
+        run: |
+          #!/bin/bash
+          aws s3 cp s3://alephzero-devnet-eu-central-1-keys-bucket/data data --recursive
+          docker run -i -v $(pwd)/data:/data --env RUST_BACKTRACE=1 --entrypoint "/usr/local/bin/aleph-node" public.ecr.aws/p6e8q1z1/aleph-node:${RELEASE_TAG} bootstrap-chain --raw --base-path /data --chain-id a0dnet1 --account-ids 5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH,5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o,5Dfis6XL8J2P6JHUnUtArnFWndn62SydeP8ee8sG2ky9nfm9,5F4H97f7nQovyrbiq4ZetaaviNwThSVcFobcA5aGab6167dK,5DiDShBWa1fQx6gLzpf3SFBhMinCoyvHM1BWjPNsmXS8hkrW,5EFb84yH9tpcFuiKUcsmdoF7xeeY3ajG1ZLQimxQoFt9HMKR,5DZLHESsfGrJ5YzT3HuRPXsSNb589xQ4Unubh1mYLodzKdVY,5GHJzqvG6tXnngCpG7B12qjUvbo5e4e9z8Xjidk3CQZHxTPZ,5CUnSsgAyLND3bxxnfNhgWXSe9Wn676JzLpGLgyJv858qhoX,5CVKn7HAZW1Ky4r7Vkgsr7VEW88C2sHgUNDiwHY9Ct2hjU8q --sudo-account-id 5F4SvwaUEQubiqkPF8YnRfcN77cLsT2DfG4vFeQmSXNjR7hD > chainspec.json
           aws s3 cp chainspec.json s3://alephzero-devnet-eu-central-1-keys-bucket/chainspec.json
 
-          # stop and clean devnet
-
+      - name: Stop and purge db Devnet
+        run: |
+          #!/bin/bash
           aws eks --region eu-central-1 update-kubeconfig --name alephzero-devnet-eu-central-1-eks
 
           kubectl delete sts aleph-node-validator -n devnet --ignore-not-found=true
           kubectl delete pvc -l app=aleph-node-validator -n devnet --ignore-not-found=true
-          kubectl delete job send-runtime-hook -n devnet --ignore-not-found=true
-
-          cd aleph-apps/aleph-node-validators/overlays/devnet/eu-central-1
-          kustomize edit set image "aleph-node-validator-image-placeholder=public.ecr.aws/p6e8q1z1/aleph-node:${COMMIT_ID}"
-          kustomize edit remove resource send-runtime-hook.yaml
-          kustomize build . | kubectl apply -f -
-          sleep 2
-          kubectl rollout status --watch --timeout=3600s statefulset/aleph-node-validator -n devnet
 
-          echo "Waiting 15 minutes"
-          sleep 900
+      - name: Start Devnet
+        env:
+          RELEASE_TAG: ${{ steps.vars.outputs.sha_short }}
+        run: |
+          #!/bin/bash
 
+          cd aleph-apps/aleph-node-validators/overlays/devnet/eu-central-1
           kustomize edit set image "aleph-node-validator-image-placeholder=public.ecr.aws/p6e8q1z1/aleph-node:${RELEASE_TAG}"
+          # we start nodes from the scratch, so we don't need to update the runtime
+          kustomize edit remove resource send-runtime-hook.yaml
```

Comment:
If we want to remove send-runtime-hook, then lets remove it from aleph-apps and ignore it all together in this workflow.

Reply:
Okay but it's non-zero probability we'll use it in the future, as we'll likely use different cloning mechanism than fork-off. So I thought just to skip it for now. With that explanation, is it okay to keep it as it is?

Reply:
Ok done, I did not understand the remark in the first place.